test/common.py

```diff
@@ -73,7 +73,8 @@ def run_tests(argv=UNITTEST_ARGS):
 NO_MULTIPROCESSING_SPAWN = os.environ.get('NO_MULTIPROCESSING_SPAWN', '0') == '1'
 TEST_WITH_ASAN = os.getenv('PYTORCH_TEST_WITH_ASAN', '0') == '1'
 TEST_WITH_UBSAN = os.getenv('PYTORCH_TEST_WITH_UBSAN', '0') == '1'
-
+BUILT_WITH_ROCM = os.getenv('PYTORCH_BUILT_WITH_ROCM', '0') == '1'
+OVERRIDE_DISABLED_UNIT_TESTS_ROCM = os.getenv('PYTORCH_OVERRIDE_DISABLED_UNIT_TESTS_ROCM', '0') == '1'
```

Comment:
I thought we decided we don't need the override?

Reply:
You also probably want to re-add the empty line you deleted.

Reply:
Yeah, I don't see why we have the OVERRIDE_DISABLED_UNIT_TESTS_ROCM flag. 

In any case that you'd ever want to run all the tests, you'd simply have to not set the env var PYTORCH_BUILT_WITH_ROCM, preventing BUILT_WITH_ROCM from being set.